src/main/java/com/google/devtools/build/lib/skyframe/LocalRepositoryLookupFunction.java

```diff
@@ -96,7 +96,15 @@ public SkyValue compute(SkyKey skyKey, Environment env)
               directory.getRoot(),
               directory
                   .getRootRelativePath()
-                  .getRelative(BuildFileName.WORKSPACE.getFilenameFragment()));
+                  .getRelative(LabelConstants.WORKSPACE_DOT_BAZEL_FILE_NAME));
+      if (!workspaceRootedFile.asPath().exists()) {
+        workspaceRootedFile =
+            RootedPath.toRootedPath(
+                directory.getRoot(),
+                directory
+                    .getRootRelativePath()
+                    .getRelative(LabelConstants.WORKSPACE_FILE_NAME));
```

Comment:
Same logic as in LocalRepositoryFunction. Can you extract to a common helper function?

Reply:
Done,